src/common/appmain.cpp

```diff
@@ -85,6 +87,15 @@ extern uartslave_t uart6slave; // PUT slave
 extern osThreadId webServerTaskHandle; // Webserver thread(used for fast boot mode)
 #endif                                 //BUDDY_ENABLE_ETHERNET
 
+void app_setup_marlin_logging();
```

Comment:
It would be better to move the function as a whole here, rather than create a forward declaration